linux/linux_reals/real_threadpool.h

```diff
@@ -18,8 +18,7 @@
         threadpool_schedule_work, \
         threadpool_timer_start, \
         threadpool_timer_restart, \
-        threadpool_timer_cancel, \
-        threadpool_timer_destroy \
+        threadpool_timer_cancel
     ) \
```

Comment:
build failed, missing `\` at the end of this line

Reply:
added